controllers/flinkcluster_reconciler.go

```diff
@@ -74,6 +74,11 @@ func (reconciler *ClusterReconciler) reconcile() (ctrl.Result, error) {
 		return ctrl.Result{}, err
 	}
 
+	err = reconciler.reconcileConfigMap()
```

Comment:
Can we move reoncileConfigMap() to the beginning so that it will be created before JM/TM deployments.